### PR TITLE
feat: manage repository permissions

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #5 
Added interface to manage repository permissions and partly implement it for yaml format settings. 
Also, had to add escaping to `Yaml2Json`.

### CODE REVIEW

File: src/main/java/com/artipie/front/misc/Yaml2Json.java [hunk 1]
```diff
@@ -25,12 +25,30 @@ public JsonStructure apply(final String yaml) {
...
+     * so we have to escape.
+     * Asterisk can be met in permissions as item of yaml sequence:
+     * Jane:
+     *   - *
+     * which means that Jane is allowed to perform any actions.
+     * And this "- *" is what we will escape.
+     * @param yaml Yaml string
+     * @return Yaml string with escaped asterisk
+     */
+    private static String escapeAsterisk(final String yaml) {
```
Comment (g4s8) [inline src/main/java/com/artipie/front/misc/Yaml2Json.java:51]: I think we can submit a bug to eoyaml about this issue. CC @amihaiemil

File: src/main/java/com/artipie/front/settings/RepoPermissions.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ */
+public interface RepoPermissions {
+
+    /**
+     * Read permissions of the repository.
+     * @param repo Repository name
+     * @return Permissions as json structure
+     * @throws com.artipie.front.api.NotFoundException If repository does not exist
+     */
+    JsonStructure get(String repo);
```
Comment (g4s8) [inline src/main/java/com/artipie/front/settings/RepoPermissions.java:22]: Why `JsonStructure` is used here? Don't we know that it's either arrau or object?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comments

File: src/main/java/com/artipie/front/misc/Yaml2Json.java (hunk 1 above)
Comment (olenagerasimova) [reply src/main/java/com/artipie/front/misc/Yaml2Json.java:51]: reported https://github.com/decorators-squad/eo-yaml/issues/493

**Comment by olenagerasimova** [comment]
@g4s8 thanks, corrected

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**